update_podspec.sh

```diff
@@ -0,0 +1,70 @@
+set -e
```

Comment:
Magic script to update podspec headers every time files are added/deleted in the library project.